views/templates/Lists/index.tpl

```diff
@@ -1,36 +1,41 @@
-<div class="main-container flex-container-main-form">
-    <h2><?= __('Create a list') ?></h2>
-
-    <div class="form-content">
-        <?= get_data('form') ?>
+<div class="main-container flex-container-full">
+    <h2><?= __('Lists') ?></h2>
+    <div class="create-list-wrapper">
+        <div class="xhtml_form">
+            <form method="post" id="createList" name="createList" action="/taoBackOffice/Lists">
+                <input type="hidden" class="global" name="createList_sent" value="1">
+                <div class="form-toolbar">
+                    <a href="#" class="form-submitter btn-success"><span class="icon-add"></span> Create list</a>
+                </div>
+            </form>
+        </div>
     </div>
 </div>
 
 <div class="data-container-wrapper flex-container-remainder">
     <?php foreach (get_data('lists') as $i => $list): ?>
-    <section id='list-data_<?= $list["uri"] ?>' class="data-container list-container">
+    <section <?= get_data('newId') == $list['uri'] ? 'data-new-list' : '' ?> id='list-data_<?= $list["uri"] ?>' class="data-container list-container">
         <header class="container-title">
             <h6><?= _dh($list['label']) ?></h6>
         </header>
         <div class="container-content" id='list-elements_<?= $list['uri'] ?>'>
-        <ol>
-            <?php foreach ($list['elements'] as $level => $element): ?>
-            <li id="list-element_<?= $level ?>">
-                            <span class="list-element"
-                                  id="list-element_<?= $level ?>_<?= $element['uri'] ?>"><?= _dh($element['label']) ?></span>
-            </li>
-            <?php endforeach ?>
-        </ol>
-</div>
-<footer class="data-container-footer action-bar  <?php !$list['editable'] && print 'hidden'?>">
-    <?php if ($list['editable']): ?>
-    <button type="button" title="<?= __('Edit this list') ?>" class="icon-edit list-edit-btn btn-info small rgt" data-uri="<?= $list['uri'] ?>">
-    </button>
-    <button type="button" title="<?= __('Delete this list') ?>" class="icon-bin list-delete-btn btn-warning small rgt" data-uri="<?= $list['uri'] ?>">
-    </button>
-
-    <?php endif ?>
-</footer>
-</section>
+            <ol>
+                <?php foreach ($list['elements'] as $level => $element): ?>
+                <li id="list-element_<?= $level ?>">
+                    <span class="list-element" id="list-element_<?= $level ?>_<?= $element['uri'] ?>"><?=
+                        _dh($element['label']) ?></span>
```

Comment:
Maybe we can write this as a single line like it was before? 😅 
WDYT?
<details>
<summary>Option 1</summary>

```tpl
<span class="list-element" id="list-element_<?= $level ?>_<?= $element['uri'] ?>"><?= _dh($element['label']) ?></span>
```
</details>
Or store everything in a separate lines (but I'm not sure that this will render properly the element 😅)
<details>
<summary>Option 2</summary>

```tpl
<span class="list-element" id="list-element_<?= $level ?>_<?= $element['uri'] ?>">
    <?= _dh($element['label']) ?>
</span>
```
</details>

Reply:
@clagosarias already took care of this :+1: (option 1 :heavy_check_mark: ) eff24d976d8fa210617e1571ad8e4e99c4c7819a